Presentation.Web/app/components/it-contract/tabs/it-contract-tab-main.view.html

```diff
@@ -265,6 +265,25 @@ <h4>{{contract.name}}</h4>
                         </label>
                         <i ng-show="contract.isActive" class="fa fa-check" aria-hidden="true"></i>
                     </div>
+
+                    <div class="col-xs-12">
+                        <div class="form-group">
+                            <label for="last-changed-by-name" class="control-label small">Sidst redigeret: Bruger</label>
+                            <input type="text"
+                                   id="last-changed-by-name"
+                                   value="{{contract.lastChangedByName}}"
+                                   class="form-control input-sm"
+                                   disabled>
+                        </div>
+                        <div class="form-group">
+                            <label for="last-changed" class="control-label small">Sidst redigeret: Dato</label>
+                            <input type="text"
+                                   id="last-changed"
+                                   value="{{contract.lastChanged}}"
```

Comment:
lastChanged must be formatted using the common formatter used in kendo and in it-system (remember we had the same issue there)

![image](https://user-images.githubusercontent.com/13116889/174274917-f7fc33f7-a2d8-45fe-b5b0-3e597d0121a3.png)